### PR TITLE
com.iotracks to org.eclipse.iofog

### PR DESCRIPTION
**Pull request by mchepelev**

### CODE REVIEW

File: src/main/org/eclipse/iofog/core_networking/comsat_client/ComSatClient.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package main.com.iotracks.core_networking.comsat_client;
+package main.org.eclipse.iofog.core_networking.comsat_client;
```
Comment (ilaryionava) [inline src/main/org/eclipse/iofog/core_networking/comsat_client/ComSatClient.java]: same for main package part

File: pom.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>groupId</groupId>
+    <groupId>org.eclipse.iofog</groupId>
     <artifactId>ioElementCoreNetworking</artifactId>
     <version>1.0-SNAPSHOT</version>
```
Comment (ilaryionava) [inline pom.xml]: - i thinks snapshot can be removed, and 
- probably version 0.1 will be enough for now
- actifactId can be more java style like : core-networking

File: src/main/org/eclipse/iofog/core_networking/comsat_client/ComSatClientHandler.java [hunk 3]
```diff
@@ -1,9 +1,9 @@
-package main.com.iotracks.core_networking.comsat_client;
+package org.eclipse.iofog.core_networking.comsat_client;
 
 import io.netty.channel.ChannelHandlerContext;
 import io.netty.channel.SimpleChannelInboundHandler;
-import main.com.iotracks.core_networking.local_client.LocalClient;
```
Comment (ilaryionava) [inline src/main/org/eclipse/iofog/core_networking/comsat_client/ComSatClientHandler.java]: i would've change core_networking to core.networking, and local_client to local.clietn

File: src/main/org/eclipse/iofog/core_networking/main/CoreNetworking.java [hunk 4]
```diff
@@ -1,14 +1,14 @@
-package main.com.iotracks.core_networking.main;
+package org.eclipse.iofog.core_networking.main;
 
-import com.iotracks.api.IOFabricClient;
-import com.iotracks.api.listener.IOFabricAPIListener;
+import org.eclipse.iofog.api.IOFabricClient;
```
Comment (ilaryionava) [inline src/main/org/eclipse/iofog/core_networking/main/CoreNetworking.java]: needs to use latest sdk with IOFogClient, etc.

File: pom.xml (hunk 2 above)
Comment (ilaryionava) [reply pom.xml]: did you build jar file to check that build passes ?

**Review by ilaryionava — APPROVED** [review]

**Comment by ilaryionava** [comment]
@ekrylovich review, approve & merge ?

**Review by ekrylovich — APPROVED** [review]

**Pull request merged**